config/build_ornl_cades.sh

```diff
@@ -7,28 +7,36 @@
 ## * Execute this script in trunk/                            ##
 ##   ./config/build_ornl_cades.sh                             ##
 ##                                                            ##
-## Last verified: Dec 5, 2018                                 ##
+## Last verified: Mar 12, 2019                                ##
 ################################################################
 
+# module files resulting from module imports below:
+# Currently Loaded Modulefiles:
+#   1) intel/18.0.0           4) gcc/6.3.0              7) cmake/3.11.0
+#   2) openmpi/3.1.1          5) hdf5_parallel/1.10.3   8) boost/1.67.0-pe3
+#   3) PE-intel/3.0           6) fftw/3.3.5-pe3         9) libxml2/2.9.9
+
 source $MODULESHOME/init/bash
 module purge
-module load PE-intel
-module load xalt/0.7.6
-module load hdf5-parallel/1.8.17
-module load boost/1.64.0
-module load cmake/3.12.0
-module swap intel intel/18.0.0
-module swap openmpi openmpi/1.10.3
+module load PE-intel/3.0
+module load intel/18.0.0
 module load gcc/6.3.0
+module load hdf5_parallel/1.10.3
+module load fftw/3.3.5-pe3
+module load cmake
+module load boost/1.67.0-pe3
+module load libxml2/2.9.9
 module list
 
 # Set environment variables
 export FFTW_HOME=$FFTW_DIR/..
+export BOOST_ROOT=$BOOST_DIR
 
 CMAKE_FLAGS="-DCMAKE_C_COMPILER=mpicc \
              -DCMAKE_CXX_COMPILER=mpicxx \
              -DCMAKE_C_FLAGS=-xCOMMON-AVX512 \
-             -DCMAKE_CXX_FLAGS=-xCOMMON-AVX512"
+             -DCMAKE_CXX_FLAGS=-xCOMMON-AVX512 \
+             -DHAVE_MKL=1"
```

Comment:
Please use -DENABLE_MKL=1 instead of -DHAVE_MKL=1.
See the logic here #1436

Reply:
Did you find our FindMKL not working as expected even with intel compiler?

Reply:
Switched to -DENABLE_MKL=1.  Seems to work fine.

Reply:
I’m still wondering why you need it. It should not be needed when intel compiler is in use.  What happens if you don’t put it.

Reply:
Also works fine without ENABLE_MKL.  Removing it now.